src/ui/public/vis/__tests__/_agg_config.js

```diff
@@ -398,22 +398,257 @@ describe('AggConfig', function () {
       const label = aggConfig.makeLabel();
       expect(label).to.be('Count');
     });
-    it('default label should be "Percentage of Count" when Vis is in percentage mode', function () {
-      const vis = new Vis(indexPattern, {});
-      const aggConfig = vis.aggs[0];
-      aggConfig.vis.params.mode = 'percentage';
-      const label = aggConfig.makeLabel();
-      expect(label).to.be('Percentage of Count');
-    });
     it('empty label if the Vis type is not defined', function () {
       const vis = new Vis(indexPattern, {});
       const aggConfig = vis.aggs[0];
       aggConfig.type = undefined;
       const label = aggConfig.makeLabel();
       expect(label).to.be('');
     });
+    describe('format percentage', function () {
+
+      it('pointseries - default label should be "Percentage of Count" when Vis is in percentage mode', function () {
+        const vis = new Vis(indexPattern, {});
+        const aggConfig = vis.aggs[0];
+        aggConfig.vis.params = JSON.parse(`{
+        "addLegend": true,
+        "addTimeMarker": false,
+        "addTooltip": true,
+        "categoryAxes": [
```

Comment:
extract the config to fixtures ? (i think there are already some configs there) ... also i don't think the whole config is needed ?